content-node/docker-compose/.env.example

```diff
@@ -1,21 +1,28 @@
 # for a list of available config options, see
 # https://docs.earthfast.com/node-operators/content-node-setup#step-5-configure-and-run-the-content-node-container
 
-# Server configuration
-SERVER_NAME=content1.us-central1-a.testnet-sepolia.earthfastnodes.com
+
+# Server url eg `content1.us-central1-a.testnet-sepolia.earthfastnodes.com`
+SERVER_NAME=
```

Comment:
can we validate this and NODE_ID are set and SERVER_NAME doesn't have trailing slash?